### PR TITLE
Fix avatar icons in user list when using dark theme

### LINKED ISSUE #34006: [Bug]: Avatar New style looks weird
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

image  image

### Steps to reproduce

After this was implemented https://github.com/nextcloud/server/pull/33752
 

### Expected behavior

Should look more clear. Th users back end is running LDAP @CarlSchwan what do you think? 

### Installation method

_No response_

### Operating system

_No response_

### PHP engine version

_No response_

### Web server

_No response_

### Database engine version

_No response_

### Is this bug present after an update or on a fresh install?

_No response_

### Are you using the Nextcloud Server Encryption module?

_No response_

### What user-backends are you using?

- [x] Default user-backend _(database)_
- [x] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

```shell
Na
```

### Nextcloud Signing status

_No response_

### Nextcloud Logs

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by CarlSchwan**
Fix #34006

### CODE REVIEW

File: apps/settings/src/components/UserList.vue [hunk 1]
```diff
@@ -378,6 +379,10 @@ export default {
 				},
 			]
 		},
+		isDarkTheme() {
+			return window.getComputedStyle(this.$el)
+				.getPropertyValue('--background-invert-if-dark') === 'invert(100%)'
```
Comment (szaimen) [inline apps/settings/src/components/UserList.vue:384]: @skjnldsv is there a different way to check this?

Comment (CarlSchwan) [reply apps/settings/src/components/UserList.vue:384]: Currently, there isn't another way to do it in js. This is also how this is done in the avatar component

I hope that at some point we can expose a global reactive property.

**Review by szaimen — APPROVED** [review]
lgtm then but didn't test

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by andyxheli** [comment]
Hi @CarlSchwan Im not sure if i should creat a new ticker or we can reopen ticket https://github.com/nextcloud/server/issues/34006

Same results on NC 25 Beta 7 

imageimage

**Comment by nursoda** [comment]
> Same results on NC 25 Beta 7
@andyxheli You DID Shift-Reload, right? Caching seems to do odd things with the new theming…

**Comment by andyxheli** [comment]
Hey @nursoda Yes I did! Cleared cache, tried different browsers same results not sure if it matters users are populating via LDAP

**Pull request merged**